docs/release/release_0_4_14.md

```diff
@@ -0,0 +1,68 @@
+# napari 0.4.14
+
+We're happy to announce the release of napari 0.4.14!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+This is a small release containing some new features and fixing a few issues that have come up since `0.4.13` was released.
+
+This release adds the ability to convert shapes layers to labels layers in a context menu available on right clicking the layer (#3978). Our vectors layer has gained the 
+ability to render vectors in 2D across multiple slices (#2902) and points in the points
+layer can now be hidden individually (#3625).
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Complete list of changes below:
+
+## Highlights
+- Enable shapes layer (right click) > convert to labels (#3978)
+- out-of-slice rendering for Vectors (#2902)
```

Comment:
The listed changes lead with a mix of upper and lower case. Other change logs seem to consistently lead with uppercase.